infrastructure/Vagrantfile

```diff
@@ -1,8 +1,22 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-cpus = ENV['AIRY_CORE_CPUS'] || 4
-memory = ENV['AIRY_CORE_MEMORY'] || 7168
+DEFAULT_AIRY_CORE_CPUS = 4
+DEFAULT_AIRY_CORE_MEMORY = 7168
+DEFAULT_NGROK_ENABLED = "true"
+DEFAULT_AIRY_VERSION = "develop"
```

Comment:
This is not actually needed, but just so that `vagrant up` can be executed outside the bootstrap.
Currently, that fails because AIRY_VERSION is not set.